ui/StatusQ/src/StatusQ/Core/Theme/ThemePalette.qml

```diff
@@ -104,6 +104,7 @@ QtObject {
     property color white: getColor('white')
 
     property color dropShadow: getColor('black', 0.12)
+    property color dropShadow2
```

Comment:
why empty?

Reply:
It's to be overridden in light/dark palette I guess :)